lib/capybara/poltergeist/client.rb

```diff
@@ -102,14 +103,20 @@ def command
     # lose all the output from child. Process.popen can be used here and seems
     # it works with JRuby but I've experienced strange mistakes on Rubinius.
     def redirect_stdout
-      prev = STDOUT.dup
-      $stdout = @write_io
-      STDOUT.reopen(@write_io)
-      yield
-    ensure
-      STDOUT.reopen(prev)
-      $stdout = STDOUT
-      prev.close
+      if Capybara::Poltergeist.jruby?
```

Comment:
Also, I believe JRuby 9000 should support the :out option (could be wrong on that but I think so)